packages/components/src/components/PlaygroundSettings.tsx

```diff
@@ -117,46 +137,54 @@ export function generateDistributionPlotSettings(
 }
 
 // partial params for SqFnPlot.create; TODO - infer explicit type?
-export function generateFunctionPlotSettings(settings: ViewSettings) {
+export function generateFunctionPlotSettings(settings: PlaygroundSettings) {
   const xScale = SqLinearScale.create({
     min: settings.functionChartSettings.start,
     max: settings.functionChartSettings.stop,
   });
   return { xScale, points: settings.functionChartSettings.count };
 }
 
-export const ViewSettingsForm: React.FC<{
-  withFunctionSettings?: boolean;
-  fixed?: PartialViewSettings;
-  register: UseFormRegister<ViewSettings>;
-}> = ({ withFunctionSettings = true, fixed, register }) => {
-  return (
-    <div className="space-y-6 p-3 divide-y divide-gray-200">
-      <FormSection title="General Display Settings">
-        <div className="space-y-4">
-          <NumberInput
-            name="chartHeight"
-            register={register}
-            label="Chart Height (in pixels)"
-          />
-        </div>
-      </FormSection>
-
-      <DistributionViewSettingsForm fixed={fixed} register={register} />
-
-      {withFunctionSettings ? (
-        <FunctionViewSettingsForm fixed={fixed} register={register} />
-      ) : null}
+export const RenderingSettingsForm: React.FC<{
+  register: UseFormRegister<PlaygroundSettings>;
+  fixed?: PartialPlaygroundSettings;
+}> = ({ register }) => (
+  <div className="space-y-4">
+    <div className="space-y-2">
+      <NumberInput
+        name="renderingSettings.sampleCount"
+        label="Sample Count"
+        register={register}
+      />
+      <div className="mt-3">
+        <FormComment>
+          How many samples to use for Monte Carlo simulations. This can
+          occasionally be overridden by specific Squiggle programs.
+        </FormComment>
+      </div>
     </div>
-  );
-};
+    <div className="space-y-2">
+      <NumberInput
+        name="renderingSettings.xyPointLength"
+        register={register}
+        label="Coordinate Count (For PointSet Shapes)"
+      />
+      <div className="mt-2">
+        <FormComment>
+          When distributions are converted into PointSet shapes, we need to know
+          how many coordinates to use.
+        </FormComment>
+      </div>
+    </div>
+  </div>
+);
 
-export const DistributionViewSettingsForm: React.FC<{
-  register: UseFormRegister<ViewSettings>;
-  fixed?: PartialViewSettings;
+export const DistributionSettingsForm: React.FC<{
+  register: UseFormRegister<PlaygroundSettings>;
+  fixed?: PartialPlaygroundSettings;
 }> = ({ register, fixed }) => {
   return (
-    <div className="pt-8">
+    <>
```

Comment:
It's not important in this case, because this fragment has a single child and can be removed, but usually it's dangerous to export fragments from React components, because it makes them less isolated.

E.g., if you export `<><div>1</div><div>2</div></>`, the way it renders will be affected by the wrapping `<div className="space-y-4">...</div>`.

Related coding style rule that I usually follow and that was violated by the previous version: components shouldn't set their own padding/margins.